Apps/Medication/test/unit/Controllers/MedicationStatementController_Test.cs

```diff
@@ -69,10 +69,18 @@ public async Task ShouldGetMedicationStatemets()
             MedicationStatementController controller = new MedicationStatementController(authzMock.Object, svcMock.Object, httpContextAccessorMock.Object);
 
             // Act
-            RequestResult<List<MedicationStatement>> actual = (RequestResult<List<MedicationStatement>>)await controller.GetMedicationStatements(hdid);
+            IActionResult actual = await controller.GetMedicationStatements(hdid);
+
+            Assert.IsType(typeof(JsonResult), actual);
```

Comment:
lets move the assertions to under the // Verify comment

Reply:
I moved the location of the verify to include all of the assertions.